src/dashboard/src/main/migrations/0038_fix_workflow_store_dip.py

```diff
@@ -0,0 +1,189 @@
+    # 2. Modifications to the "Store DIP" chain
+    ###########################################################################
+
+    new_store_dip_chain_id = '6eb8ebe7-fab3-4e4c-b9d7-14de17625baa'
+    MicroServiceChain.objects.create(
+        id=new_store_dip_chain_id,
+        startinglink_id=store_dip_quest_link_uuid,
+        description='Do not upload DIP')
+
+    MicroServiceChainChoice.objects.create(
+        id='6513c071-9b99-4e7c-b0c7-8792ff9cb273',
+        choiceavailableatlink_id=upload_dip_choice_link_uuid,
+        chainavailable_id=new_store_dip_chain_id)
+
+    ###########################################################################
+    # 3. Remove the unused links and their dependencies
+    ###########################################################################
+
+    MicroServiceChainLink.objects.get(
+        id=dstry_retrieve_locat_link_uuid).delete()
+    MicroServiceChainLink.objects.get(
+        id=dstry_store_dip_locat_link_uuid).delete()
+    MicroServiceChainLink.objects.get(
+        id=dstry_store_dip_link_uuid).delete()
+    MicroServiceChainLink.objects.get(
+        id=dstry_move_uploaded_link_uuid).delete()
+
+    ###########################################################################
+    # 4. Change "Completed" link to "Handle unstored DIP"
+    ###########################################################################
+
+    completed_link = MicroServiceChainLink.objects.get(
+        id=completed_link_uuid)
+    completed_tc = completed_link.currenttask
+    completed_tc.description = 'Handle unstored DIP'
+    completed_tc.save()
+    completed_stc = StandardTaskConfig.objects.get(id=completed_stc_uuid)
+    completed_stc.arguments = ('"%SIPDirectory%" '
+                               '"%rejectedDirectory%." '
+                               '"%watchDirectoryPath%uploadedDIPs/"')
+    completed_stc.execute = 'handleUnstoredDIP_v0.0'
+    completed_stc.save()
+
+    ###########################################################################
+    # 5. Change "move" to "copy" in "move to uploadedDIPS dectory"
+    ###########################################################################
+
+    # By using copy_v0.0, the MCPServer will know where to look for the
+    # processing config XML file, viz. in the uploadDIP/ directory.
+    # UPDATE StandardTasksConfigs SET execute = 'copy_v0.0', arguments = '"%SIPDirectory%" "%watchDirectoryPath%uploadedDIPs/." -R' WHERE pk = '302be9f9-af3f-45da-9305-02706d81b742';
+    move_uploaded_dips_stc_uuid = '302be9f9-af3f-45da-9305-02706d81b742'
+    StandardTaskConfig.objects.filter(id=move_uploaded_dips_stc_uuid).update(
+        execute='copy_v0.0',
+        arguments='"%SIPDirectory%" '
+                  '"%watchDirectoryPath%uploadedDIPs/." '
+                  '-R')
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [('main', '0037_mediaconch_policy_checks')]
+    operations = [migrations.RunPython(data_migration)]
```

Comment:
This is a clear style deviation from Django's suggested way, not that I care but I find interesting that you prefer your way over the convention :D